src/test/regress/sql/ref_citus_local_fkeys.sql

```diff
@@ -0,0 +1,130 @@
+\set VERBOSITY terse
```

Comment:
Should we add some isolation tests as well?

Reply:
given that we're already going through the standard process utility, there is really not much we can test additionally for Citus specific. 

Can you think of anything?